src/openrct2/network/NetworkBase.h

```diff
@@ -32,7 +31,8 @@ class NetworkBase
     bool Init();
     void Close();
     uint32_t GetServerTick();
-    void Update();
+    // FIXME: This is currently the wrong function to override in System, will be refactored later.
```

Comment:
(On https://github.com/OpenRCT2/OpenRCT2/pull/15232/commits/b3bb2f5d80765bf383fd60d5bd5b2c79fd0dcbba)

Will this be fixed still in this PR or later? If later, it would help to detail why it's the wrong function

Reply:
Needs to be fixed later, the naming is inconsistent when it comes to Update/Tick. Update will be called every frame where Tick is fixed at 40hz. Thats why its wrong right now, its not an actual Update.